### PR TITLE
No docker openpilot sim for testing modifications

### PR DESCRIPTION
**Pull request by creikey**
Useful for locally testing UI stuff without needing to build a docker image

### CODE REVIEW

File: tools/sim/start_openpilot.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+cd $DIR
+
+cd ../../
+
+export PASSIVE="0"
```
Comment (grekiki) [inline tools/sim/start_openpilot.sh:8]: Is this necessary?

Comment (creikey) [reply tools/sim/start_openpilot.sh:8]: Without them I get communication errors

**Comment by adeebshihadeh** [comment]
There's no need for a new script. Just add the necessary environment variables to `tmux_script.sh`.

**Comment by creikey** [comment]
> There's no need for a new script. Just add the necessary environment variables to `tmux_script.sh`. `launch_openpilot.sh` already sets up the environment. `tmux_script.sh` should already do what you want.

```sh
./bridge.py 
creikey@pc:~/openpilot/tools/sim$ ./bridge.py 
Traceback (most recent call last):
  File "./bridge.py", line 11, in <module>
    import cereal.messaging as messaging
ModuleNotFoundError: No module named 'cereal'
```
```sh
./launch_openpilot.sh
creikey@pc:~/openpilot/tools/sim$ ./launch_openpilot.sh
Traceback (most recent call last):
  File "./manager.py", line 9, in <module>
    import cereal.messaging as messaging
ModuleNotFoundError: No module named 'cereal'
creikey@pc:~/openpilot/tools/sim$
```

Starting from the tmux script does not properly set the `PYTHONPATH` as is done in `start_openpilot_docker.sh:14`

```sh
  EXTRA_ARGS="-v $PWD/../..:/root/openpilot -e PYTHONPATH=/root/openpilot:$PYTHONPATH"
```

**Comment by adeebshihadeh** [comment]
All of our tools kind of assume that PYTHONPATH is already set to your openpilot directory, but I'd merge a PR to set it in `tools/sim/launch_openpilot.sh`.

**Comment by creikey** [comment]
> All of our tools kind of assume that PYTHONPATH is already set to your openpilot directory, but I'd merge a PR to set it in `tools/sim/launch_openpilot.sh`.

I'll just set it in my bashrc, and `bridge.py` is started with no script anyways. Maybe this should be added to the sim wiki? Something like

> You can run the simulator on locally edited files by first setting your pythonpath (from the `tools/sim` directory):

`export PYTHONPATH="$(cd ../../ && pwd)"`

> Then by running `./tmux_script.sh`

**Pull request closed without merging**